src/Entity/CustomerOptions/CustomerOption.php

```diff
@@ -191,7 +191,7 @@ public function setConfiguration(array $configuration): void
     {
         // Setting the new values
         foreach ($configuration as $key => $value) {
-            $optionKey                                = str_replace('_', '.', $key);
+            $optionKey                                = str_replace(':', '.', $key);
```

Comment:
Is this intentional?

Reply:
Well when using _ this conflicts with the Symfony's block naming convention. This is a bug fix.

Reply:
:+1: